### PR TITLE
add1 program (C)

### PR DESCRIPTION
**Pull request by WasshoiSugi**
おなしゃす！

### CODE REVIEW

File: kadai.c [hunk 1]
```diff
@@ -0,0 +1,70 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <math.h>
+
```
Comment (puyogo-suzuki) [inline kadai.c:4]: Use constant value. like below.
`
#define STUDENT_COUNT 20
`

File: kadai.c [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+				sci[i] = sci[j];
+				sci[j] = tmp1;
+			}
+			if(eng[i] < eng[j]){
+				tmp2 = eng[i];
+				eng[i] = eng[j];
+				eng[j] = tmp2;
+			}
+		}
+	}
```
Comment (puyogo-suzuki) [inline kadai.c]: Use qsort function.
`
void qsort( void * data , size_t data_cnt
       , size_t data_size
       , int( * func )( const void * , const void * );
`

File: kadai.c [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		stu_hensati[0][i] = (sci[i] - sci_ave) / (sci_hensa) * 10 + 50;
+	}
+	for(i=0;i<20;i++){
+		stu_hensati[1][i] = (eng[i] - eng_ave) / (eng_hensa) * 10 + 50;
+	}
+
+	printf("~偏差値~\n");
+	for(i=0;i<20;i++){
+		printf("~生徒%d~ 理科: %.2f, 英語: %.2f\n", i+1, stu_hensati[0][i], stu_hensati[1][i]);
+}
```
Comment (puyogo-suzuki) [inline kadai.c]: Please sort both of 偏差値 and 得点.

**Review by puyogo-suzuki — COMMENTED** [review]
Please write more CLEAR code!
* Use Standard Library.
* Use Constant Value.
* Create function for common routines.

**Comment by WasshoiSugi** [comment]
関数化はだるいお。

**Comment by puyogo-suzuki** [comment]
Okay... the import things is not creating more clear code...

**Comment by WasshoiSugi** [comment]
偏差値は個人的にそーとしなくてもいいかなと。

**Comment by WasshoiSugi** [comment]
#define使いました

**Comment by puyogo-suzuki** [comment]
Okay, let's improve it in the future commits.

**Comment by puyogo-suzuki** [comment]
Yup, Ok!

**Pull request merged**